public/main.css

```diff
@@ -644,12 +650,7 @@ tbody {
 }
 
 #download-progress {
-  margin: 0 auto;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  flex-direction: column;
-  text-align: center;
+  width: 590px;
```

Comment:
Is this fixed `width` going to work for mobile?
I didn't see it overridden below for any mobile media queries.